esx_service/utils/kvESX.py

```diff
@@ -166,35 +168,45 @@ def delete(volpath):
     return True
 
 
+# Align a given string to the specified block boundary.
+def align_str(kv_str, block):
+   # Align string to the next block boundary.
```

Comment:
nit: please add "-1" is to accommodate extra \n" to the comments - I was really confused by the -1 until I read the follow up